packages/two_dimensional_scrollables/lib/src/tree_view/tree.dart

```diff
@@ -940,9 +948,7 @@ class _TreeViewState<T> extends State<TreeView<T>>
           controller.forward();
         case false:
           // Collapsing
-          controller.reverse().then((_) {
-            _unpackActiveNodes();
-          });
+          controller.reverse();
```

Comment:
hwo about change below code
https://github.com/flutter/packages/blob/bb53e5d536aeba377b5f8877ad735973293ae200/packages/two_dimensional_scrollables/lib/src/tree_view/tree.dart#L936-L945
to

```
      switch (node._expanded) {
        case true:
          // Expanding
          controller.forward();
        case false:
          // Collapsing
          controller.reverse();
      }
```